### DISCUSSION TITLE
github actions的status显示queued

### DISCUSSION THREAD

**Discussion by Johntang666** [tiddly-gittly/TidGi-Desktop#394] [Q&A]
# github actions同步之后，访问失败。
后面发现还在queued中，status并没有sucess
![image](https://github.com/tiddly-gittly/TidGi-Desktop/assets/67173358/73859582-cf0e-4cfc-94e7-e3464b3657e7)

**Comment by linonetwo** [comment]
免费的东西，慢点正常。充钱了（企业版）就快了
